test/sales_engine_test.rb

```diff
@@ -112,4 +148,15 @@ def test_acceptance_item
     assert_equal '75107', engine.item_repository.items.first.unit_price
     assert_equal 'Item Autem Minima', engine.item_repository.items[1].name
   end
+
+  def test_acceptance_customer
+    # skip
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.